src/test/java/io/crate/jmx/integrationtests/MetricsITest.java

```diff
@@ -113,7 +101,7 @@ public void testThreadPoolsMetrics() {
     @Test
     public void testCircuitBreakersMetrics() {
         List<String> names = Arrays.asList(
-                "parent", "query", "jobs_log", "operations_log", "fielddata", "request", "in_flight_requests");
+                "parent", "query", "jobs_log", "operations_log", "request", "in_flight_requests");
```

Comment:
Aren't the fielddata breaker stats still exposed?

Reply:
They are deprecated and return the default value -1:
https://github.com/crate/crate/commit/9d57c83a5624587d422539f6d2a0267958621926

Reply:
Should they stay here for now ?

Reply:
I think we should try to keep newer jmx_exporters working with all versions in the current major release series. So every `4.x` version should work with any recent jmx_exporter release.

Reply:
Ok, thanks for the feedback. I adapted the `CrateCollectorTest` that it works with all 4.x CrateDB versions, and the `MetricsITest` that it works with `latest-nightly`. All deprecated outdated metrics are added back. So this change only affects only the tests.